### DISCUSSION TITLE
Questions about the LingQ API

### DISCUSSION THREAD

**Discussion by PseudoSloth** [thags/lingqAnkiSync#31] [Q&A]
Hi there,
I'm asking here as your addon documentation provides the best LingQ API documentation that I could find 🙈
Do you know if it's possible to get a list of known words via the API? Or how to you find the list of known words?
I've tried exporting the .csv from their UI, but it only includes words in the vocabulary lists - or is this what you do too?

Thanks a mil!

**Comment by thags** [comment]
You would probably need to [use this](https://github.com/thags/lingqAnkiSync?tab=readme-ov-file#1-get-lingqs-from-a-language) to get all of the lingqs for the language, and then do something custom to filter/sort only the items with a lingq status of known. 
I think the known status is when "status" and "extendedStatus" are both 4.

**Reply by Money961** [reply]
Hi, give: 'LingqPK' what would it be?
